docs/cli/index.md

```diff
@@ -1293,6 +1294,49 @@ starport tools protoc -- --version
 
 * [starport tools](#starport-tools)	 - Tools for advanced users
 
+## starport tools completions
+
+Generates a completion script. 
+
+**Synopsis**
+
+The ``completions`` command generates a completion script to be used in your shell. However, the generated script depends on
+[bash-completion](https://github.com/scop/bash-completion) being installed and enabled in your system. Most of *nix like
+operating systems come with it by default and we assume it is already installed and operational. 
+
+```bash
+starport tools completions [bash|zsh|fish|powershell]
+```
+
+**Examples**
+
+```
+starport tools completions bash
+```
+
+**Options**
+
+```
+-h, --help   help for completions
+```
+
+### Enable starport autocompletion
+
+In order to enable the autocompletion every time you open a terminal, we present two solutions :
```

Comment:
```suggestion
Use one of these methods to enable the autocompletion each time you open a terminal:
```

Reply:
I think there is also one innacuracy here... This could induce the user to repeat the process whenever it opens the terminal. 

I think it could be something like :

"If you wish the autocompletion to be enabled at the moment you start a terminal, use one of those methods: "

What do you think, @barriebyron ? :)

Reply:
a ha! (where is the lightbulb emoji?) I see how it works now, I didn't review the commands careful. 

we can just tell the user to enable it and then give them the "enable now" and "persist" options. See how these suggested changes work.

Reply:
I made the suggested change above ^^ line 1325